torch/_subclasses/fake_tensor.py

```diff
@@ -819,8 +821,10 @@ def __init__(
         allow_non_fake_inputs=False,
         shape_env=None,
         static_shapes=None,
+        allow_unsafe_data_ptr_access=True,
```

Comment:
Just checking: if this flag exists basically just to only ban data ptr access in PT2, but the long term plan is to unconditionally ban data ptr access on FakeTensor, do we want to mark this argument private so we can more easily kill it later?

Reply:
hmm

Reply:
Yeah that sounds reasonable